bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveIndicatorCommands.java

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
I think we discussed earlier that this class is not needed? If so, please remove it - it doesn't appear to be used and it's completely empty... (or am I missing something?)